### PR TITLE
Use new is_terse and terse functions

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: experimental/Schemes/WeilDivisor.jl [hunk 1]
```diff
@@ -608,7 +608,7 @@ generated by rational functions ``f₁,…,fᵣ ∈ K(X)``.
 end
 
 function Base.show(io::IO, L::LinearSystem)
-  if get(io, :supercompact, true)
+  if isterse(io)
```
Comment (joschmitt) [inline experimental/Schemes/WeilDivisor.jl]:
```suggestion
  if is_terse(io)
```

File: src/Rings/orderings.jl [hunk 2]
```diff
@@ -1719,7 +1723,7 @@ function induce(vars::Vector{Int}, o::ProdOrdering)
 end
 
 @doc raw"""
-    induce(vars::AbstractVector{<:MPolyRingElem}, ord::ModuleOrdering)
+    induce(vars::AbstractVector{<:MPolyRingElem}, ord::MonomialOrdering)
```
Comment (joschmitt) [inline src/Rings/orderings.jl]: There are also some bits of #3679 in here?

**Comment by lgoettgens** [comment]
The printing part of the dev docs need to be adapted as well; just applying your script led to some inconsistencies.

File: experimental/GModule/GaloisCohomology.jl [hunk 3]
```diff
@@ -1390,7 +1390,7 @@ end
 function Base.show(io::IO, m::MIME"text/plain", a::RelativeBrauerGroupElem)
   io = pretty(io)
   print(io, "Element of relative Brauer group of ", Lowercase(), parent(a).k)
-  io = IOContext(io, :supercompact => true, :compact => true)
+  io = terse(io)
```
Comment (lgoettgens) [inline experimental/GModule/GaloisCohomology.jl:1393]: This no longer setting `:compact` makes the lhs of the below list print verbose, as https://github.com/thofma/Hecke.jl/blob/master/src/NumFieldOrd/NfOrd/Ideal/Ideal.jl#L148 does not use `is_terse`. There are some more occurrences in this file still using `:compact` and thus producing weird results.
This is the reason for failing doctests.

Comment (fingolfin) [reply experimental/GModule/GaloisCohomology.jl:1393]: I'll add the `compact` back here, and then will have a look at replacing uses of `compact` in Hecke (so we can drop `compact` here in a future PR)

File: src/Rings/orderings.jl (hunk 2 above)
Comment (fingolfin) [reply src/Rings/orderings.jl]: Ouch! Good catch, not sure how that happened -- hopefully managed to remove all traces now.

**Comment by fingolfin** [comment]
CI is green now

**Review by joschmitt — APPROVED** [review]
Looks good to me

**Comment by fingolfin** [comment]
I've also changed the references to `supercompact` in the documentation to `terse`.

The only references to `supercompact` left are in tests where `sprint `or `repr` are called with `context = :supercompact => true`. I will provide add a replacement function to AA we can use to adjust this kind of test.

**Comment by fingolfin** [comment]
See https://github.com/Nemocas/AbstractAlgebra.jl/pull/1701

**Review by lgoettgens — APPROVED** [review]

**Pull request merged**